src/vs/workbench/contrib/accessibility/browser/accessibleView.ts

```diff
@@ -471,38 +492,37 @@ export class AccessibleView extends Disposable {
 	}
 
 	disableHint(): void {
-		if (!this._currentProvider) {
+		if (!this._currentProvider || !('verbositySettingKey' in this._currentProvider)) {
 			return;
 		}
 		this._configurationService.updateValue(this._currentProvider?.verbositySettingKey, false);
 		alert(localize('disableAccessibilityHelp', '{0} accessibility verbosity is now disabled', this._currentProvider.verbositySettingKey));
 	}
 
-	private _updateContextKeys(provider: IAccessibleContentProvider, shown: boolean): void {
+	private _updateContextKeys(provider: AccessibleViewContentProvider, shown: boolean): void {
 		if (provider.options.type === AccessibleViewType.Help) {
 			this._accessiblityHelpIsShown.set(shown);
 			this._accessibleViewIsShown.reset();
 		} else {
 			this._accessibleViewIsShown.set(shown);
 			this._accessiblityHelpIsShown.reset();
 		}
-		if (provider.next && provider.previous) {
+		if ('next' in provider && 'previous' in provider && provider.next && provider.previous) {
```

Comment:
Having an `isXYZ` method here instead of `'next' in provider` would mean that you don't need to check for `'previous'` too.